theano/compat/six.py

```diff
@@ -1,6 +1,6 @@
 """Utilities for writing code that runs on Python 2 and 3"""
 
-# Copyright (c) 2010-2015 Benjamin Peterson
+# Copyright (c) 2010-2016 Benjamin Peterson
```

Comment:
This isn't our code so the year should only change when (if) we import a new version of six.